aws_batch/pipeline.py

```diff
@@ -357,6 +359,44 @@ def generate_taxid_annotated_m8(input_m8, output_m8, accession2taxid_db):
             outf.write(new_line)
     outf.close()
 
+def read_columnfile_into_list(file_name):
+    with open(file_name) as f:
+        content = f.readlines()
+    return [x.strip("\n") for x in content]
```

Comment:
rstrip or strip? 
a simpler way would be 
return [x for x in f] 
directly. 

Can you also explain what's read_columnfile_into_list? it's not immediately obvious

Reply:
I thought that was the part you wanted me to extract into a function in one of your previous comments. But since it can be changed into a single line, I think I'll remove this function

Reply:
improved function and left it in as read_file_into_list